tasks/main.yml

```diff
@@ -1,6 +1,12 @@
 ---
 # tasks file for ncats_webd
 
```

Comment:
> For example, in this case the removal of the greenlet pin magically appears in the merge commit but it should really be in its own commit. See https://github.com/cisagov/ansible-role-cyhy-archive/pull/50 for an example of what I'm talking about.

I try to do that(see https://github.com/cisagov/ansible-role-cobalt-strike/pull/79).  I remember making a commit for greenlet, but I think I ended up starting that PR from scratch because of an issue I was having and must've done it at the same time the second go round

Reply:
> Let's go ahead and remove these unnecessary lines:

I saw that in another repo but wasn't sure if the extra line got deleted accidentally.  I'll make sure I remove them in other repos I see them in as well